### ISSUE TITLE
Make logging more consistent

### ISSUE THREAD

**Issue by derneuere** [LibrePhotos/librephotos#170]
We currently have the problem, that we don't apply the different logging levels consistently. That means that the user has currently to log more information than he needs to. We should improve that.

**Comment by gawel** [comment]
There is others problems. It looks like nothing is made to work outside a docker context.

I'm trying to use librephotos on a bare server. I've add problems with thumbnails and waited for https://github.com/LibrePhotos/librephotos/pull/168 to be merge to have a deeper look and see if it fixes my problem.

Today I've updated my install and got another problems about logs dir. The app is trying to log to a file in /logs ( https://github.com/LibrePhotos/librephotos/blob/dev/api/util.py#L17 ) but this is total non-sense in a non-docker installation.

So my question is: do you guys plan to only support docker installation ? or do you need some helps to make it works outside docker ?

**Comment by derneuere** [comment]
Yes you are right. This isn't yet implemented see: https://github.com/LibrePhotos/librephotos/issues/57

You can look here: https://github.com/LibrePhotos/librephotos/pull/179 and here https://github.com/LibrePhotos/librephotos-linux/pull/1

The pull request is untested. Would be great if you could test them and share your ideas for improving the experience outside of docker.

**Comment by gawel** [comment]
Nice. Didn't see this one coming. I've made my own ansible playbook so I'll not use the install script but I'll try to test the branch. Thanks for the pointer

**Comment by derneuere** [comment]
We should also combine the logs into one file and rename it to librephotos.log
